puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/rpm_packaging.groovy

```diff
@@ -1,78 +1,19 @@
-def packages_to_build
-
```

Comment:
Did you intend to edit this file? This does not appear used by the job.

Reply:
Uh, I honestly have no idea why this is showing up here, I didn't change it, but it appears to be this way in the repo, and when I try to check it out from origin nothing changes.